fiftyone/core/annotation/generate_label_schemas.py

```
@@ -0,0 +1,431 @@
+            if values and len(values) <= foac.CHECKBOXES_OR_RADIO_THRESHOLD:
+                settings[foac.COMPONENT] = (
+                    foac.CHECKBOXES if is_list else foac.RADIO
+                )
+
+            elif len(values) <= foac.VALUES_THRESHOLD:
+                settings[foac.COMPONENT] = foac.DROPDOWN
+
+            if settings[foac.COMPONENT] in foac.VALUES_COMPONENTS:
+                settings[foac.VALUES] = values
+
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)